docs/Contributing/Releasing-Fleet.md

```diff
@@ -17,6 +17,7 @@ Note: Please prefix versions with `fleet-v` (eg. `fleet-v4.0.0`) in git tags, He
    - [package.json](https://github.com/fleetdm/fleet/blob/main/tools/fleetctl-npm/package.json) (do not yet `npm publish`)
    - [Helm chart](https://github.com/fleetdm/fleet/blob/main/charts/fleet/Chart.yaml) and [values file](https://github.com/fleetdm/fleet/blob/main/charts/fleet/values.yaml)
    - [Terraform variables](https://github.com/fleetdm/fleet/blob/main/infrastructure/dogfood/terraform/aws/variables.tf)
+   - Kubernetes manifest file
```

Comment:
Actually this should be updated to be a link

Reply:
Okay. do you mean I should have the manifest file in the Fleet repo and link to it?

Reply:
yeah, exactly

Reply:
Got you! Can you point me to where in the Fleet repo I should add the manifest file?

Reply:
Oh nevermind, instead since this is inline it should probably link to `docs/Deploying/Server-Installation.md`'s `## Deploying Fleet on Kubernetes`

Reply:
Oh, that makes sense. Making that change now

Reply:
Added!